### DISCUSSION TITLE
How to make winget reload its settings?

### DISCUSSION THREAD

**Discussion by lackovic** [microsoft/winget-cli#2979] [Q&A] [answered]
I ran `winget settings` and added:

```js
    "experimentalFeatures": {
        "pinning": true,
    },
```

After saving, when I try to pin a package with `winget pin add --id "<Package.Id>"` I get:

```powershell
This feature is a work in progress, and may be changed dramatically or removed altogether in the future. To enable it, edit your settings ('winget settings') to include the experimental feature: 'pinning'
```

How can I make winget reload the settings? Rebooting the computer doesn't work.

Using Windows Package Manager v1.4.10173.

**Comment by Trenly** [comment, marked as answer]
Pinning functionality is not present in version 1.4.10173, even though the commands for it are.

This is because https://github.com/microsoft/winget-cli/pull/2733 was merged into the codebase for release 1.4 and https://github.com/microsoft/winget-cli/pull/2813 wasn't implemented until version 1.5.

In order to use Pinning you will need at least version [1.5.441-preview](https://github.com/microsoft/winget-cli/releases/tag/v1.5.441-preview)

cc @denelon for future process improvements and communications, if necessary
